apps/gidd/views.py

```diff
@@ -883,6 +883,34 @@ def _export_disaggregated_excel(self, qs):
                 self.string_join(EXTERNAL_ARRAY_SEPARATOR, item.locations_type),
                 self._get_displacement_occurred(item.displacement_occurred),
             ])
+
+        # Tab 2
+        ws2 = wb.create_sheet('2_Context_Displacement_data')
+        ws2.append([
+            'ISO3',
+            'Year',
+            'Figure cause',
+            'Figure category',
+            'Description',
+            'Figures',
+            'Figures rounded',
+        ])
+        pfa_qs = PublicFigureAnalysisFilterSet(
+            data=self.request.query_params,
+            queryset=PublicFigureAnalysis.objects.all()
+        ).qs.order_by('iso3', 'year')
```

Comment:
```suggestion
        ).qs.order_by('iso3', 'year', 'id')
```

Let's add `id` here too.